packages/bridge/src/bridges/index.ts

```diff
@@ -0,0 +1,115 @@
+
+
+import { NetworkConfig } from '@arcadeum/provider'
+import { BigNumberish, providers } from 'ethers'
+
+export type MoveEstimate = {
+  crossTime: number,
+  steps: number
+}
+
+export type Move = {
+  transactionHash: string,
+  completeTx: providers.TransactionRequest[] | undefined,
+  isCompleted: boolean,
+  isPending: boolean,
+  fromChainId: number,
+  toChainId: number
+}
+
+export type MoveNative = Move & {
+  amount: BigNumberish
+}
+
+export type MoveERC20 = Move & {
+  token: string,
+  amount: BigNumberish
+}
+
+export type MoveERC721 = Move & {
+  token: string,
+  id: BigNumberish
+}
+
+export type MoveERC1155 = Move & {
+  token: string,
+  ids: BigNumberish[],
+  amounts: BigNumberish[]
```

Comment:
Thinking about it, there may be some bridges that use the `data` field for ERC-1155 transfers.